### ISSUE TITLE
Agents can't 'see' empty dirs

### ISSUE THREAD

**Issue by JacobFV** [getcursor/cursor#693]
1) If you can, please include a screenshot of your problem

![Screenshot from 2023-07-20 14-15-00](https://github.com/getcursor/cursor/assets/40343913/b66f5a34-ace9-4ef4-96e1-23d4bc6fb41d)

2) Please include the name of your operating system

Ubuntu 22.04

3) If you can, steps to reproduce are super helpful

Ask it to create a file in an empty dir that you previously created. It cannot see the dir

**Comment by JacobFV** [comment]
Similar error with extra closing parenthesis.

**Comment by JacobFV** [comment]
I don't think this would be a problem if the LLM was reprompted with the updated contents whenever an extension performed text injections like autoclosing parentheses or tags

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
